src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -109,6 +117,23 @@ public CommentsTag(final Github ghub) {
                 )
             );
             Logger.info(this, "duplicate tag %s commented", tag);
+        } else if (VersionValidator.getInstance().isVersionValid(tag)
+            && !VersionValidator.getInstance().isReleaseValid(tag, priors)) {
+            issue.comments().post(
+                    String.format(
+                            CommentsTag.PHRASES.getString(
+                                    "CommentsTag.version-too-low"
+                            ),
+                            tag,
+                            Collections.max(priors)
+                    )
+            );
+            Logger.info(
+                    this,
+                    "tag %s must be greater than previous version %s",
```

Comment:
The other logging statements don't use localization strings, so I am just keeping it similar to the existing implementation, in this example.